### DISCUSSION TITLE
Navigate by week?

### DISCUSSION THREAD

**Discussion by jk171505** [gpbl/react-day-picker#1659] [Ideas]
Is this possible to navigate by weeks instead of months? Meaning, the visible range is 7 days and click on prev/next arrows moves the visible range by 7 days, not the whole month.

**Comment by gpbl** [comment]
It is not possible at the moment. Interesting use case, tho... do you have an example of this kind of implementations?

**Comment by jk171505** [comment]
Here: https://github.com/hurkanyakay/react-weekly-day-picker
Pretty old, but shows exactly what I mean.

**Comment by gpbl** [comment]
Oh, this kind of view is not between the goals of this component. Thanks for the suggestion!
